packages/image_picker/image_picker_android/android/src/main/java/io/flutter/plugins/imagepicker/ImagePickerDelegate.java

```diff
@@ -29,12 +29,6 @@
 import java.util.Map;
 import java.util.UUID;
 
-enum CameraDevice {
```

Comment:
Fixes a warning about referencing a class other that the main class (I forget the actual term in the warning) from another file.